intelmq/bots/parsers/shadowserver/parser.py

```diff
@@ -133,12 +133,11 @@ def parse_line(self, row, report):
                     try:
                         value = conv_func(raw_value)
                     except Exception:
+                        """ fail early and often in this case. We want to be able to convert everything """
                         self.logger.error('Could not convert shadowkey: %r, '
                                           'value: %r via conversion function %r.',
                                           shadowkey, raw_value, conv_func.__name__)
-                        value = None
-                        # """ fail early and often in this case. We want to be able to convert everything """
-                        # self.stop()
+                        raise
```

Comment:
These are cases where the data is unexpected or can't be handled otherwise. The same happens when there is no conversion function. What we actually do here is, to add more context with the logging a few lines above.